src/store/bridge/actions.ts

```diff
@@ -290,7 +299,7 @@ async function updateBridgeProxyLockedBalance(context: ActionContext<any, any>):
   const { networkSelected, networkType } = rootState.web3;
 
   if (address && networkSelected && networkType) {
-    const bridgeApi = getters.bridgeApi as typeof evmBridgeApi | typeof subBridgeApi;
+    const bridgeApi = getBridgeApi(context) as typeof evmBridgeApi | typeof subBridgeApi;
```

Comment:
Do we need `as` here? It should already have this type

Reply:
Yes, to exlude `ethBridgeApi`  type (this is the only one place where i need to do it)